public/app/plugins/panel/graph/Legend/LegendSeriesItem.tsx

```diff
@@ -4,7 +4,8 @@ import { TimeSeries } from 'app/core/core';
 import { SeriesColorPicker, SeriesIcon } from '@grafana/ui';
 import { selectors } from '@grafana/e2e-selectors';
 
-export const LEGEND_STATS = ['min', 'max', 'avg', 'current', 'total'];
+export const LEGEND_STATS = ['min', 'max', 'avg', 'current', 'total'] as const;
```

Comment:
Maybe we could use enum values: https://github.com/grafana/grafana/blob/408bc06bab5efa2d5432a70c3d8079c7e2ef9ae5/packages/grafana-data/src/transformations/fieldReducer.ts#L7 instead?